src/UCTSearch.cpp

```diff
@@ -284,17 +284,17 @@ bool UCTSearch::is_running() const {
 }
 
 int UCTSearch::est_playouts_left() const {
-    auto elapsed_millis = now() - m_start_time;
+    auto elapsed_millis = now() - m_time_after_initial_playouts;
     auto playouts = m_playouts.load();
     if (!Limits.dynamic_controls_set() && !Limits.movetime) {
         // No time control, use playouts or visits.
         const auto playouts_left =
                 std::max(0, std::min(m_maxplayouts - playouts,
                                      m_maxnodes - m_root->get_visits()));
         return playouts_left;
-    } else if (elapsed_millis < 1000 || playouts < 100) {
-        // Until we reach 1 second or 100 playouts playout_rate
-        // is not reliable, so just return max.
+    } else if (elapsed_millis < 10 || playouts < 10) {
+        // Until we reach 10 millisecond and 10 playouts playout_rate
```

Comment:
you changed the comment from "or" to "and" but the logic remains ||, not &&

Reply:
a or b == !(a and b)

Reply:
Yes, the existing comment was wrong.